torchvision/datasets/_stereo_matching.py

```diff
@@ -14,6 +14,9 @@
 from .utils import _read_pfm, download_and_extract_archive, verify_str_arg
 from .vision import VisionDataset
 
+T1 = Tuple[Image.Image, Image.Image, Optional[np.ndarray], np.ndarray]
```

Comment:
Nice way to keep things simple

Reply:
Yeah. I wonder if we can make the name more expressive though. Not blocking, but `T1` and `T2` are rather opaque.

Reply:
I agree but I tried to think about a meaning naming and honestly I didn't manage to find any :sweat_smile: